src/transformers/image_utils.py

```diff
@@ -195,19 +195,20 @@ def resize(self, image, size, resample=PIL.Image.BILINEAR, torch_resize=False, m
                 The size to use for resizing the image. If `size` is a sequence like (h, w), output size will be
                 matched to this.
 
-                If `size` is an int and `torch_resize` is `False`, then image will be resized to (size, size). If
-                `size` is an int and `torch_resize` is `True`, then smaller edge of the image will be matched to this
-                number. i.e, if height > width, then image will be rescaled to (size * height / width, size).
+                If `size` is an int and `default_to_square` is `True`, then image will be resized to (size, size). If
+                `size` is an int and `default_to_square` is `False`, then smaller edge of the image will be matched to
+                this number. i.e, if height > width, then image will be rescaled to (size * height / width, size).
             resample (`int`, *optional*, defaults to `PIL.Image.BILINEAR`):
                 The filter to user for resampling.
-            torch_resize (`bool`, *optional*, defaults to `False`):
-                Whether or not to replicate `torchvision.transforms.Resize` with support for resizing only the smallest
-                edge and providing an optional `max_size`.
+            default_to_square (`bool`, *optional*, defaults to `True`):
+                If set to `False`, will replicate
```

Comment:
I would add a first sentence: "How to convert `size` when it is a single int." Then "If set to `True`, the `size` will be converted to a square `(size,size)`." and then the docstring currently there.